nipy/algorithms/registration/groupwise_registration.py

```diff
@@ -506,17 +494,17 @@ def adjust_subsampling(speedup, dims):
 def single_run_realign4d(im4d,
                          affine_class=Rigid,
                          time_interp=True,
-                         loops=LOOPS,
-                         speedup=SPEEDUP,
-                         borders=BORDERS,
-                         optimizer=OPTIMIZER,
+                         loops=5,
+                         speedup=5,
+                         refscan=0,
```

Comment:
Alexis - I think you've changed the order of parameters, in that `refscan` is now after `speedup` not after `maxfun`.   That's an API change (in case anyone is using say `borders` as a positional argument.  Any reason for that?